src/main/java/com/pinterest/secor/common/KafkaClient.java

```diff
@@ -141,8 +141,11 @@ private Message getMessage(TopicPartition topicPartition, long offset,
             payloadBytes = new byte[payload.limit()];
             payload.get(payloadBytes);
         }
+        Long timestamp = null;
+        timestamp = messageAndOffset.message().timestamp();
```

Comment:
There will be a back compatibility issue here, either because:
1. People still use kafka 8 library
2. Or people on kafka 10 but don't want to use the timestamp from kafka message (they still want to retrieve the timestamp from other fields).

I think we need to introduce a config param, something like useKafkaTimestamp, and it should be false by default.